modules/nw-infw-operator-rules-object.adoc

```diff
@@ -148,8 +149,9 @@ spec:
          ports: 22
      action: Allow
    - order: 20
-     action: Deny <3>
+     action: Deny <4>
 ----
 <1> Multi-interface cluster
-<2> `0.0.0.0/0` set to match any CIDR
-<3> `action` set to `deny`
+<2> The value should be set to `true` and the label must match the `nodeSelector` label that you specified in the `ingressfirewallconfig` CR.
```

Comment:
It should be matchLabels matching key value pair in label. The value may not be true e.g. node label would be as follows:-
Label on node
<pre>
labels:
    beta.kubernetes.io/arch: amd64
    beta.kubernetes.io/os: linux
    kubernetes.io/arch: amd64
    kubernetes.io/hostname: worker-0
    kubernetes.io/os: linux
    node-fw: icmp
    node-role.kubernetes.io/worker: ""
    node.openshift.io/os_id: rhcos
</pre>

The firewall rule is as follows:-
<pre>
oc get ingressnodefirewall ingressnodefirewall-icmp -n openshift-ingress-node-firewall -oyaml
apiVersion: ingressnodefirewall.openshift.io/v1alpha1
kind: IngressNodeFirewall
metadata:
  creationTimestamp: "2023-12-06T18:46:16Z"
  generation: 1
  name: ingressnodefirewall-icmp
  resourceVersion: "1355836"
  uid: a942bf69-646c-4588-9da6-4ecf03b3f43d
spec:
  ingress:
  - rules:
    - action: Deny
      order: 3
      protocolConfig:
        icmp:
          icmpType: 8
        protocol: ICMP
    sourceCIDRs:
    - 10.128.2.47/24
  interfaces:
  - genev_sys_6081
  nodeSelector:
    matchLabels:
      node-fw: icmp
status:
  syncStatus: Synchronized
</pre>

Reply:
@JoeAldinger This comment is valid for line number 86 as well. Just want to point out matching value need not be 'true' only.

Reply:
@asood-rh what is the difference between a value of `true` and `false` here do?